### PR TITLE
[rascal] Export config interfaces

### PR DESCRIPTION
**Pull request by qlonik**
Just exporting configuration interfaces so that it is easier to write wrappers of provided functions.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

**Review by MartinTechy — APPROVED** [review]

**Comment by qlonik** [comment]
I just noticed that there is an error on `subscribeAll` function: definitions declare that filter has to be a string type, but according to docs and source code, it seems that it is supposed to be a function returning boolean. Should I fix it in this pr or in another?

https://github.com/guidesmiths/rascal#subscribe-all

**Comment by MartinTechy** [comment]
> I just noticed that there is an error on `subscribeAll` function: definitions declare that filter has to be a string type, but according to docs and source code, it seems that it is supposed to be a function returning boolean. Should I fix it in this pr or in another?
> 
> https://github.com/guidesmiths/rascal#subscribe-all

Correct, I think it is better to create a separate PR. Don't forget to add tests for the change so we can merge it without any DT maintainer approval.

**Pull request merged**
